.github/workflows/automated-tests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x]
```

Comment:
No new version beyond 20 to test against?

Reply:
No, release date of 21.x is 2023-10-17 :slightly_smiling_face: